.ci/DEB/cpeditor.desktop

```diff
@@ -0,0 +1,16 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=CP Editor
+GenericName=Competitive Programming Editor
+GenericName[ru]=Редактор кода для спортивного программирования
+GenericName[zh_CN]=竞争编程编辑器
+Exec=/usr/bin/cpeditor
+Comment=CP Editor is a Qt-based, lightweight and cross-platform code editor specially designed for competitive programming.
+Comment[RU]=CP Editor - это легкий кросс-платформенный редактор кода на базе Qt, специально разработанный для спортивного программирования.
+Comment[zh_CN]=CP Editor 是一个基于 Qt 的轻量级跨平台代码编辑器，专为算法竞赛设计。
+Icon=/usr/share/icons/cpeditor.png
```

Comment:
File extension not required.

It's better to have only one desktop file for Linux. Please modify the already existing file at ci/linux/cpeditor.desktop

Reply:
I think that extension (.png) is required:
![image](https://user-images.githubusercontent.com/32099652/90419137-2d384e80-e0c7-11ea-9abd-34efe213a681.png)

Reply:
Read https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html

You need to only specify the name of the icon-file. The extension (png) is added automatically. It is not good idea to have an absolute path to icon file because this way when user changes its theme/style the icon will not change.